test/game_manager_test.exs

```diff
@@ -0,0 +1,149 @@
+defmodule GameManagerTest do
+  use ExUnit.Case
+  import Mock
+  import GameManager
+  import ExUnit.CaptureIO
+  doctest GameManager
+
+  @invalid_input_message "You entered an invalid move. Please try again.\n"
+
+  test "it can prompt the current player to make a move" do
```

Comment:
This test is really only testing the mock, since the mock is responsible for returning the string asserted agains

Reply:
Also like we talked about, we should be mindful of the scope of our mocks. At an integration level, are we mocking too much? Can we exercise more of our code under test by reducing the scope of our mocks?